src/types/schema/invites.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize, Clone, Copy, PartialEq, PartialOrd, Eq, Ord)]
+/// Query parameters for routes that list invites.
+/// # Reference:
+/// Read: <https://discord-userdoccers.vercel.app/resources/invite#invite-structure>
```

Comment:
Minor change to make the docs prettier and afaik this only for the one `Get Invite` route

```suggestion
/// Query parameters for the `Get Invite` route.
///
/// # Reference:
/// Read: <https://docs.discord.sex/resources/invite#query-string-params>
```